### PR TITLE
feat: added Google Tag Manager

### PR DESCRIPTION
**Pull request by FroudeDescartes**
@rubystream 
I have successfully installed [GTM ](https://nuxt.com/modules/nuxt-gtm) but I am not sure how to proceed.
Is it activated globally now, and running as soon as the landing page, or any other page loads?

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -34,6 +35,32 @@ export default defineNuxtConfig({
   routeRules: {
     '/api/search.json': { prerender: true },
   },
+  // Google Tag Manager
+  runtimeConfig: {
+    public: {
+      gtm: {
+        id: 'GTM-xxxxxx',
```
Comment (rubystream) [inline nuxt.config.ts:42]: I think that the real GTM ID and auth parameters should be provided.

**Review by rubystream — APPROVED** [review]
LGTM

**Comment by rubystream** [comment]
AFAIK the loading page event should be triggered from the start. 
There is a possibility to emit custom events as well as stated in documentation.

File: nuxt.config.ts (hunk 1 above)
Comment (FroudeDescartes) [reply nuxt.config.ts:42]: Yes, this is obtained from the personal profile on GTM.

**Comment by FroudeDescartes** [comment]
Great, thanks. I will mark this as Done.

**Pull request merged**